src/bundle/Resources/public/scss/_inputs.scss

```diff
@@ -22,11 +28,8 @@
         border-color: $ibexa-color-primary;
     }
 
-    &:active {
-        border-color: $ibexa-color-dark;
-    }
-
-    &:focus {
+    &:focus,
+    &.ibexa-input--text.form-control:focus {
```

Comment:
Is form-control class needed here?

Reply:
I added this because I needed to overwrite bootstrap styles defined with selector:
`form:not(.form-inline) .form-control:focus`